demos/classification_benchmark_demo/cpp_gapi/README.md

```diff
@@ -0,0 +1,177 @@
+# Classification Benchmark C++ G-API Demo
+
+![](../classification_benchmark.gif)
+
+The demo visualizes OpenVINO performance on inference of neural networks for image classification.
+
+## How It Works
+
+On startup, the application reads command line parameters and loads a classification model to OpenVINO™ Runtime plugin for execution. It might take some time for demo to read all input images. Then the demo performs inference to classify the images and places them on grid.
+
+The demo starts in "Testing mode" with fixed grid size. After calculating the average FPS result, it will switch to normal mode and grid will be readjusted depending on model performance. Bigger grid means higher performance. You can repeat testing by pressing "Space" or "R" button.
+
+When "ground truth" data is applied, the color coding for the text, drawn above each image, shows whether the classification was correct: green means correct class prediction, red means wrong.
+
+You can stop the demo by pressing "Esc" or "Q" button. After that, the average metrics values will be printed to the console.
+
+> **NOTE**: By default, Open Model Zoo demos expect input with BGR channels order. If you trained your model to work with RGB order, you need to manually rearrange the default channels order in the demo application or reconvert your model using the Model Optimizer tool with the `--reverse_input_channels` argument specified. For more information about the argument, refer to **When to Reverse Input Channels** section of [Embedding Preprocessing Computation](@ref openvino_docs_MO_DG_Additional_Optimization_Use_Cases).
```

Comment:
Do we support `--reverse_input_channels` flag properly?

Reply:
That's model optimizer's flag, not demo's one

Reply:
https://github.com/openvinotoolkit/open_model_zoo/blob/master/demos/classification_benchmark_demo/cpp/main.cpp#L233

Reply:
Your point is valid overall, I just wanted to point out that the piece of text which you started this conversation under shouldn't be modified because it relates to Model Optimizer, not the demo